e2e-identity/tests/utils/fmk.rs

```diff
@@ -123,8 +122,8 @@ impl E2eTest {
         self.display_step("get a nonce for creating an account");
         let mut resp = self.client.execute(req).await?;
         self.display_resp(Actor::AcmeServer, Actor::WireClient, Some(&resp));
-        resp.expect_status_ok()
-            .expect_header("cache-control", "no-store")
+        resp.expect_status(StatusCode::OK)
```

Comment:
Above (line 102) you replaced `expect_status_ok` with `expect_status_success`, but here you replaced the same call with `expect_status(StatusCode::OK)`. If there's no particular reason for it, I'd prefer to be consistent and use `expect_status_success` here as well.